chirp/drivers/tdh8.py

```diff
@@ -1024,6 +1025,22 @@ class TDH8(chirp_common.CloneModeRadio):
                  chirp_common.PowerLevel("Mid",  watts=4.00),
                  chirp_common.PowerLevel("High", watts=8.00)]
 
+    def check_set_memory_immutable_policy(self, existing, new):
```

Comment:
Does this need to be on the base class or can it be only on the RT730 subclass? The failure is because you didn't add it to the whitelist in `test_chirp_common` but if we add it on the base class here, we have to add *all* the subclasses to the whitelist as well, which would be less than ideal, unless it's really required.

Reply:
Actually it should probably be in the RT730 and any other model that supports airband (the TD-H3). I actually had put them there but switched it back to where it is now since I was unable to get it to work.

Reply:
Okay, so it doesn't work for some reason when put on the RT730 itself? It really should...

Reply:
I guess not if it isn't in the whitelist. I will add it to the whitelist. Should I just do it for the RT730 and don't worry about the TD-H3 unless someone complains about it?

Reply:
I dunno, maybe just do it for the RT730 now so we can get this merged. So you're going to move this to the RT730 subclass and add just that to the whitelist right?

Not sure if it's unclear, but the whitelist only affects the test passing, not the runtime operation...

Reply:
Moving to RT730 and adding to whitelist passes the tests.